packages/web/utils/coin.ts

```diff
@@ -1,15 +1,20 @@
 import { CoinPretty } from "@keplr-wallet/unit";
 
 /** Combines coins of the same denom into the same CoinPretty object by sum. */
-export function aggregateCoinsByDenom(coins: CoinPretty[]): CoinPretty[] {
+export function aggregateCoinsByDenom<TCoin extends CoinPretty>(
+  coins: TCoin[]
```

Comment:
I suggest we omit the `T` since we shouldn't mind if it's a generic or not
```suggestion
  coins: Coin[]
```